### ISSUE TITLE
New Super Mario Bros. (EUR) Black screen entering "Minigames"

### ISSUE THREAD

**Issue by bell07** [ahezard/nds-bootstrap#621]
The game "New Super Mario Bros. (EUR)" works fine, but if I call the "Minigames", I get Black screens. I can only return to the Home using home button.
Same rom file works fine using desmume emulator on desktop.

- [X]  New 3DS/2DS (XL)
- [X]  Luma3DS
- [X]  nds-bootstrap Release v0.19.1 / TWLMenu++v8.1.1

Game: New Super Mario Bros. (EUR)

### Expected behaviour
Minigames part playeable

### Actual behaviour
Both screens black if launching the Minigames part

**Comment by MasterGamingYT** [comment]
This is due to soft resetting being broken, the game tries to restart the system (I think), but fails, resulting in a black screen

**Comment by bell07** [comment]
Thank you for the hint. I see my issue is duplicate of #358 and depends on #259

**Comment by MasterGamingYT** [comment]
No problem

**Comment by MasterGamingYT** [comment]
However, SM64DS's minigames do work, so until soft resetting is patched, we could play some of the same minigames on SM64DS

**Comment by MasterGamingYT** [comment]
Got that from issue #259
